src/wallet/walletdb.cpp

```diff
@@ -878,7 +879,6 @@ DBErrors WalletBatch::LoadWallet(CWallet* pwallet)
     } catch (...) {
         result = DBErrors::CORRUPT;
     }
-    m_batch->CloseCursor();
```

Comment:
In f2c83f4b:

In case of read completion or an exception, the cursor will stay open up to the next return statement if we don't call `cursor.reset()` here.

Reply:
Done

Reply:
The cursor is within the `try` scope, so when we get here, it will already be out of scope and destroyed.

Reply:
true, I missed that. nvm then.